src/backend/aspen/app/views/sample.py

```diff
@@ -84,6 +84,28 @@ def _format_gisaid_accession(
     return {"status": "no_info", "gisaid_id": None}
 
 
+def _format_lineage(sample: Sample):
+    pathogen_genome = sample.uploaded_pathogen_genome
+    if pathogen_genome:
+        lineage = {
+            "lineage": pathogen_genome.pangolin_lineage,
+            "probability": pathogen_genome.pangolin_probability,
+            "version": pathogen_genome.pangolin_version,
+            "last_updated": api_utils.format_date(
+                pathogen_genome.pangolin_last_updated
+            ),
+        }
+    else:
+        lineage = {
+            "lineage": None,
+            "probability": None,
+            "version": None,
+            "last_updated": None,
+        }
```

Comment:
Can you add an issue that we need to handle this for `CalledConsensusGenomes`?

Reply:
Sure will do, in that case will we be getting the lineage info from a `CalledPathogenGenome`?

Reply:
does [this](https://app.clubhouse.io/genepi/story/136767/lineage-info-for-when-we-have-a-called-consensus-genome-instead-of-an-uploaded-pathogen-genome) work?

Reply:
> Sure will do, in that case will we be getting the lineage info from a `CalledPathogenGenome`?

Yes.

> does [this](https://app.clubhouse.io/genepi/story/136767/lineage-info-for-when-we-have-a-called-consensus-genome-instead-of-an-uploaded-pathogen-genome) work? 

Perfect!